### ISSUE TITLE
Multiple input context merge

### ISSUE THREAD

**Issue by andreyrikunov** [RoRu/Video-Packager#11]
In practical scenarios there are two types of input context at least: 

- user context (e.g. input/output file path)
- system/script level context (e.g. passlogfile path)

The user context is used to solve practical user (better to say - clients, in relation to available workflows) tasks having a certain properties of a workflow, while the system context is used by video engineers who are responsible to manage quality of a workflow.

In other words there is a need to merge multiple input context into composite one before particular workflow run. Merge should not be limited by the number of 2 context.
